tests/providers/google/cloud/sensors/test_gcs.py

```diff
@@ -72,7 +73,7 @@ def context():
     """
     Creates an empty context.
     """
-    context = {"data_interval_end": datetime.utcnow()}
+    context = {"data_interval_end": timezone.utcnow()}
     yield context
```

Comment:
I just tried, no tests using this fixture actually seem to need this key. Let’s try to just remove it and see what happens?

Reply:
Some test uses this fixture, however all this tests also work fine if instead of this context provide just `{}`

Reply:
Yep CI green after I remove this fixture and replace it everywhere by `{}`